src/transaction/RewardSplit.tsx

```diff
@@ -26,8 +26,9 @@ const RewardSplit: React.FC<RewardSplitProps> = ({ txData }) => {
     : BigNumber.from(0);
 
   const minerReward = paidFees.sub(burntFees);
-  const burntPerc =
-    Math.round(burntFees.mul(10000).div(paidFees).toNumber()) / 100;
+
+  // for depositTx, paidFees == 0 so ignore
+  const burntPerc = paidFees != BigNumber.from(0) ? 0 : Math.round(burntFees.mul(10000).div(paidFees).toNumber()) / 100;
```

Comment:
This line should not work as respected.

Reply:
Fixed via https://github.com/testinprod-io/otterscan/pull/5